source/apache-way/apache-project-maturity-model.md

```diff
@@ -103,22 +102,21 @@ formats that are expected to stay readable in the long term.
 
 <dt id="RE20">RE20</dt>
 <dd>
-Releases are approved by the project's PMC (see CS10), in order to make them an act of the Foundation. 
+The project's Project Management Committee (PMC) (see CS10) approves each software release in order to make the release an act of the Foundation. 
 </dd>
 
 <dt id="RE30">RE30</dt>
 <dd>
-Releases are signed and/or distributed along with digests that can be reliably used to validate the downloaded archives. 
+Releases are signed and/or distributed along with digests that anyone can be reliably use to validate the downloaded archives. 
```

Comment:
Releases are signed, and distributed along with digests and signatures that anyone can reliably use to validate the downloaded archives.